sample.env

```diff
@@ -172,6 +172,12 @@ MAINTENANCE_WINDOW=
 # Button can be disabled by setting the value to blank
 REPORT_ISSUE_URL=https://github.com/bigbluebutton/greenlight/issues/new
 
+# The link to the Need help? button that appears on the Account Dropdown
+#
+# Defaults to the Greenlight documentation
+# Button can be disabled by setting the value to blank
+HELP_URL=http://docs.bigbluebutton.org/install/greenlight-v2.html
```

Comment:
Can you change this to https://docs.bigbluebutton.org/greenlight/gl-overview.html please

Reply:
done